asset/style/site.scss

```diff
@@ -28,49 +41,51 @@ $c-yellow-light: lighten($c-yellow, 35%);
 // Site Styles
 // -----------
 
-// Type tweaks just for better reading
-body {
-	line-height: 1.4;
-}
-pre {
-	overflow: auto;
-}
-a {
-	color: $c-blue-dark;
+@import "typography";
+
+// Clearfix mixin
+@mixin clearfix {
+	&:after {
+		content: " ";
+		display: block;
+		clear: both;
+	}
 }
-a:visited {
-	color: $c-purple-dark;
+.clearfix {
+	@include clearfix;
 }
 
 // Page width/positioning
 .page {
 	max-width: 700px;
 	margin: 0 auto;
-	padding: 20px;
+	padding: $spacing;
 }
 
 // Construction notice
 .construction-notice {
-	margin-bottom: 20px;
-	padding: 10px;
+	margin-bottom: $spacing;
+	padding: $spacing-small;
 	background-color: $c-yellow-light;
 }
 .construction-notice__content {
 	margin: 0;
-	font-size: 14px;
+	font-size: $font-size-small;
 }
 
 // Site header
 .site-header {
-	margin-bottom: 40px;
-	padding-bottom: 10px;
-	border-bottom: 2px dotted #ccc;
+	margin-bottom: $spacing-large;
+	padding-bottom: $spacing-small;
+	border-bottom: $spacing-rule dotted $c-grey-light;
 }
 
 // Site branding
-.site-brand {
+.site-brand,
+.site-brand:visited {
 	display: block;
 	line-height: 30px;
```

Comment:
I thought about it, but then it's only in one place and it's really arbitrary because using the standard spacing looked weird 😄